src/Sulu/Component/Content/Document/Behavior/BlameBehavior.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+/**
+ * Adds the user who created and lastly changed the document.
+ */
+interface BlameBehavior
+{
+    /**
+     * @return UserInterface
+     */
+    public function getCreator();
```

Comment:
Is this information persisted? Do we actually restore the `UserInterface` instance that created a document when loading a document from PHPCR?